Code/Common/include/sitkImage.h

```diff
@@ -193,20 +193,27 @@ namespace simple
     /** \brief Get the number of components for each pixel
      *
      * For scalar images this methods returns 1. For vector images the
```

Comment:
Possibly rephrase:
For images with scalar or complex pixel types this method returns one. For images with vector pixel type the method returns the number of vector components.

Reply:
done